### DISCUSSION TITLE
Dot path for colors not working with TS 5.0.2

### DISCUSSION THREAD

**Discussion by midrizi** [nandorojo/dripsy#280] [Q&A] [answered]
Hi @nandorojo,

Seems like not all the issues are fixed with the latest release `v4.0.0` and upgrading TS to `5.0.2`

With TypeScript `4.4.4` it does work:
![ts_4.4.4](https://user-images.githubusercontent.com/56638730/231400790-085e0124-3116-4051-9e8e-fc57f2ff6948.png)

But in `5.0.2` it doesn't:
![ts_5.0.2](https://user-images.githubusercontent.com/56638730/231401311-3326fa31-f6e5-400d-aa30-e69d0b2ec8f9.png)

**Comment by nandorojo** [comment]
can i see your theme?

**Reply by nandorojo** [reply]
I didn't realize people were using dot notation colors, I'll have to look into adding this

**Comment by nandorojo** [comment, marked as answer]
Fixed in 4.1.0: `yarn add dripsy`

Release: https://github.com/nandorojo/dripsy/releases/tag/v4.1.0
Commit: https://github.com/nandorojo/dripsy/commit/e8272cd46261a34554c16fd370a536c5514e1312

**Comment by midrizi** [comment]
@nandorojo 

Wow that was quick, thanks a lot.
